src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,18 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        bulldozer.stopWork();
+
+        Excavator excavator = new Excavator();
+        excavator.doWork();
+        excavator.stopWork();
+
+        Truck truck = new Truck();
+        truck.doWork();
+        truck.stopWork();
```

Comment:
Посмотри чеклист: Use abstract reference instead of specific one where possible: https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#use-abstract-reference-instead-of-specific-one-where-possible . (Было бы круто создать масив из машин и запускать их и останавливать в цикле)